src/Field.tsx

```diff
@@ -3,15 +3,33 @@ import * as React from 'react';
 
 import { FormikProps } from './formik';
 
+export interface FieldProps {
+  component: keyof React.ReactHTML | React.ComponentType<any>;
+  name: string;
+  type?: string;
```

Comment:
What about the placeholder, autoComplete, etc?

Reply:
The `Field` component only need this props to work. 
The placeholder, autoComplete, ... definitions should be in the CustomInputProps, no ?